src/components/Kanban/Board.tsx

```diff
@@ -58,7 +58,7 @@ export default function KanbanBoard<T extends { id: string }>(
       </div>
       <DragDropContext onDragEnd={props.onDragEnd}>
         <div className="h-full overflow-scroll" ref={board}>
-          <div className="flex items-stretch px-0 pb-2">
+          <div className="flex items-stretch px-0 pb-2 containerss">
```

Comment:
Tailwind allows for custom breakpoints for one-off cases, let's use that instead of creating a new class.

https://tailwindcss.com/docs/responsive-design#using-custom-breakpoints Check the very last section.

Reply:
Remove the custom class @gurramkarthiknetha 

Use the existing breakpoints in Tailwind instead.